### PR TITLE
Reword to clarify the intent of external access

### LINKED ISSUE #6887: Issue with docs/setup/getting-started/index.md
https://istio.io/docs/setup/getting-started/#confirm 

Command should be:
`curl http://$GATEWAY_URL/productpage`

As it stands currently with `echo`, the command does not illustrate successful accessibility from the outside into the application.

### PR DESCRIPTION
**Pull request by shamsher31**
To fix #6887

### CODE REVIEW

File: content/en/docs/setup/getting-started/index.md [hunk 1]
```diff
@@ -327,11 +327,10 @@ $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -
 
 ### Verify external access {#confirm}
 
-Confirm that the Bookinfo application is accessible from outside. Copy the
-output of this command and paste into your browser:
+Confirm that the Bookinfo application is accessible from outside.
```
Comment (frankbu) [inline content/en/docs/setup/getting-started/index.md]:
```suggestion
Confirm that the Bookinfo application is accessible from outside
by viewing the Bookinfo product page using a browser.

1. Run the following command to retrieve the external address of the Bookinfo application.

    {{< text bash >}}
    $ echo http://$GATEWAY_URL/productpage
    {{< /text >}}
    
1. Paste the output from the previous command into your web browser and confirm that the Bookinfo product page is displayed.
```

Comment (shamsher31) [reply content/en/docs/setup/getting-started/index.md]: Looks good, Also adding the output of the `echo` to describe how the output will like.

`http://192.168.99.100:32194/productpage`

Comment (shamsher31) [reply content/en/docs/setup/getting-started/index.md]: Thanks @frankbu

File: content/en/docs/setup/getting-started/index.md [hunk 2]
```diff
@@ -327,12 +327,17 @@ $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -
...
+by viewing the Bookinfo product page using a browser.
 
-{{< text bash >}}
-$ echo http://$GATEWAY_URL/productpage
-{{< /text >}}
+1.  Run the following command to retrieve the external address of the Bookinfo application.
+
+    {{< text bash >}}
+    $ echo http://$GATEWAY_URL/productpage
+    http://192.168.99.100:32194/productpage
```
Comment (frankbu) [inline content/en/docs/setup/getting-started/index.md]: I'm not sure whether adding example output here is helpful or not. I wonder if somebody will read the next step ("Paste the output from the previous command") and will then copy the displayed output above and try it. It's providing one more way of making a stupid mistake.

Comment (shamsher31) [reply content/en/docs/setup/getting-started/index.md]: I agree I think it's not useful and can cause a potential mistake from the user.

**Comment by ericvn** [comment]
@shamsher31 You'll probably need to push another commit to get the `netlify` test to run again (retest won't work). That said, I'm trying to get a PR merged to fix the failing lint tests (hopefully will get merged in 20 minutes or so). Once #6886 gets merged, try a change and hopefully things will pass and merge here to.

**Pull request merged**